README.md

```diff
@@ -1,2 +1,22 @@
 # zap-cli
-A Node.js CLI command for searching for, or opening a YouTube video with Zen Audio Player.
+
+[![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/zen-audio-player/zen-audio-player.github.io?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
+[![Build Status](https://travis-ci.org/zen-audio-player/zap-cli?branch=master)](https://travis-ci.org/zen-audio-player/zap-cli)
```

Comment:
The first link here should be `https://api.travis-ci.org/zen-audio-player/zap-cli.svg?branch=master`, emphasis on the `.svg` which sets up the image link